README.md

```diff
@@ -100,6 +100,24 @@ roslaunch psen_scan_v2 psen_scan_v2.launch sensor_ip:=192.168.0.10 host_ip:=192.
 ```
 This example configures the safety laser scanner at 192.168.0.10 to send it´s frames to 192.168.0.20:3050.
 
+## TF Frames
+The location of the TF frames is shown in the image below.
+Note that these names defined by the `prefix` parameter.
+This can be changed with aforementioned launchfile parameters.
+For example when you are running multiple scanners.
+<p align="center">
+<img src="img/frames.png" width="800px" alt="PILZ safety laser scanner frames" title="frames">
+</p>
```

Comment:
I think side and rear view already give all necessary information for this topic. Maybe omit the other two at this point?

Reply:
I like them for clarity